hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -548,6 +553,8 @@ public HRegionServer(Configuration conf, CoordinatedStateManager csm)
     this.numRetries = this.conf.getInt(HConstants.HBASE_CLIENT_RETRIES_NUMBER,
         HConstants.DEFAULT_HBASE_CLIENT_RETRIES_NUMBER);
     this.threadWakeFrequency = conf.getInt(HConstants.THREAD_WAKE_FREQUENCY, 10 * 1000);
+    this.compactionCheckFrequency = conf.getInt(PERIOD_COMPACTION, this.threadWakeFrequency);
+    this.flushCheckFrequency = conf.getInt(PERIOD_FLUSH, this.threadWakeFrequency);
```

Comment:
For keeping the old behavior, if both new paras are unset, then just use threadWakeFrequency as their default value.